projects/wave-core-new/src/lib/time/time-slider/time-slider.component.html

```diff
@@ -0,0 +1,14 @@
+<div #timeline id="timeline" class="visualization"></div>
+<div fxLayout="row" fxLayoutGap="0.5rem" fxLayoutAlign="space-between baseline" appearance="fill">
+    <button mat-raised-button (click)="centerTimeline()">Center</button>
+    <mat-select
+        placeholder="Timescale"
+        [(ngModel)]="selectedScale"
+        [(value)]="selectedScale"
+        (selectionChange)="changeScale(selectedScale)"
+    >
+        <mat-option *ngFor="let scale of timeScale" [value]="scale.value">
+            {{ scale.text }}
+        </mat-option>
+    </mat-select>
```

Comment:
How does this scale differ from https://github.com/geo-engine/geoengine-ui/blob/main/projects/wave-core-new/src/lib/project/project.service.ts#L323-L328 ?